docs/additional_examples.md

```diff
@@ -0,0 +1,212 @@
+# Additional Examples
+Examples come from [RCSB PDB Data API documentation](https://data.rcsb.org/#examples)
+
+### Entries
+Fetch information about structure title and experimental method for PDB entries:
+```
+{
+  entries(entry_ids: ["1STP", "2JEF", "1CDG"]) {
+    rcsb_id
+    struct {
+      title
+    }
+    exptl {
+      method
+    }
+  }
+}
+```
+```python
+from rcsbapi.data import Query
+query = Query(input_ids={"entry_ids": ["1STP","2JEF","1CDG"]},input_type="entries", return_data_list=["entries.rcsb_id", "struct.title", "exptl.method"])
```

Comment:
For this and other examples, I would suggest cleaning up the indenting so it's all visible without scrolling left to right:

```suggestion
query = Query(
    input_ids={"entry_ids": ["1STP", "2JEF", "1CDG"]},
    input_type="entries",
    return_data_list=["entries.rcsb_id", "struct.title", "exptl.method"]
)
```